besu/src/main/java/org/hyperledger/besu/RunnerBuilder.java

```diff
@@ -854,6 +863,7 @@ public Runner build() {
                 dataDir,
                 besuController.getProtocolManager().ethContext().getEthPeers());
     methods.putAll(besuController.getAdditionalJsonRpcMethods(jsonRpcApis));
+    methods.putAll(rpcEndpointServiceImpl.getPluginMethods(jsonRpcConfiguration.getRpcApis()));
```

Comment:
It looks like we will override built-ins here

Reply:
Last one wins :smile: 

Happy to put some checks in - if people think it's the right behaviour to stop people overriding existing methods.